scripts/buck_setup.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+$PROXY_CONFIG = $1
```

Comment:
Isn't it how it supposed to be?
```suggestion
PROXY_CONFIG = "$1"
```